server/events/output_updater.go

```diff
@@ -62,28 +61,6 @@ type ChecksOutputUpdater struct {
 }
 
 func (c *ChecksOutputUpdater) UpdateOutput(ctx *command.Context, cmd PullCommand, res command.Result) {
-
-	if res.Error != nil || res.Failure != "" {
```

Comment:
does this mean we only update status if command result is a success?

Reply:
Nope we only update output for project level commands, not the aggregate command regardless of success or failure because it is handled by the command runners.  

In the plan command runner, we make this status update at the end of the operation: https://github.com/lyft/atlantis/blob/8cf99abfd260c9da2ab0db5cf9aa940e896de4be/server/events/plan_command_runner.go#L194
which creates a checkrun like this: https://github.com/lyft/orchestration-sandbox/pull/1032/checks?check_run_id=7789536260